### DISCUSSION TITLE
User name

### DISCUSSION THREAD

**Discussion by salimdeerhold** [community/community#74551] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

before commit message smahrjan is shown( smahrjan 'commit message') in github, in my opinion this is the username. But the main problem is that is not the user name i have created and is not thte one that is shown in my github profile.
Could some body help me with this?

**Comment by ktauchathuranga** [comment]
is your Git settings are correct?
type `git config -l`
and check if the details right or wrong

**Reply by salimdeerhold** [reply]
I need to check emal and user name right?

**Reply by salimdeerhold** [reply]
Thanks

**Comment by EliasAfara** [comment, marked as answer]
Check out your **author information in Git**:
   - GitHub uses the Git version control system for managing repositories. The username and email associated with your commits in a Git repository are determined by your local Git configuration.
   - Ensure that your local Git configuration has the correct username and email set. You can check this by running the following commands in your project directory:

     ```bash
     git config user.name
     git config user.email
     ```

     If these are not set correctly, you can update them with:

     ```bash
     git config --global user.name "Your Name"
     git config --global user.email "your-email@example.com"
     ```

**Reply by salimdeerhold** [reply]
Thanks
